src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -792,7 +792,8 @@ namespace OpenLoco::ObjectManager
     // is possible and if not permutates the name until it is valid.
     static fs::path findObjectPath(std::string& filename)
     {
-        auto objPath = Environment::getPath(Environment::PathId::objects);
+        // Name will always be replaced
+        auto objPath = Environment::getPath(Environment::PathId::objects) / "1.DAT";
```

Comment:
Should `1.DAT` be a constant somewhere, so it can easily be changed? (Presumably, it is replaced somewhere?)

Reply:
It's immediately replaced a few lines later. Tbh i think I'll redesign the function it's a bit messy